src/App.tsx

```diff
@@ -2,26 +2,45 @@ import React from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = React.useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const getAllGoods = () => getAll().then(setGoods);
+  const get5FirstGoods = () => get5First().then(setGoods);
+  const getRed = () => getRedGoods().then(setGoods);
```

Comment:
Seems that you repeat the same logic here. Would be great to use some common handler. For example: 
```
const fetchAndSetGoods = (fetchFunction) => {
  return fetchFunction().then(setGoods);
};